src/App.tsx

```diff
@@ -1,7 +1,21 @@
-import React from 'react';
+import { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export const App: React.FC = () => {
+  const [keyPressed, setKeyPressed] = useState('');
+
+  useEffect(() => {
+    document.addEventListener('keydown', (event: KeyboardEvent) => {
+      setKeyPressed(event.key);
+    });
+  }, []);
```

Comment:
don't forget to cleanup before component unmounts - a callback after `return` statement

```suggestion
const keyupHandler = (event: KeyboardEvent) => {
      setKeyPressed(event.key);
    };

  useEffect(() => (
    document.addEventListener('keyup', keyupHandler)
    
    return () => document.removeEventListener('keyup', keyupHandler);
  ), []);
```